core-libs/setup/recipes/b2b/b2b-storefront.module.ts

```diff
@@ -5,7 +5,6 @@ import {
   provideDefaultConfig,
   provideDefaultConfigFactory,
 } from '@spartacus/core';
-import { OrganizationModule } from '@spartacus/my-account/organization';
```

Comment:
Should we still have dependency on `@spartacus/my-account` in setup lib?
It's no longer referenced.

Reply:
👍